MANIFEST.in

```diff
@@ -1,9 +1,10 @@
 include *.txt
 recursive-include wof/flask/templates *
-recursive-include wof/examples/flask/odm2/timeseries *
+recursive-include wof/examples *
 prune wof/examples/flask/barebones*
 prune wof/examples/flask/csv_server*
 prune wof/examples/flask/csv_tutorial*
 prune wof/examples/flask/odm_1_1*
+prune test
```

Comment:
I am also removing the test files from the source tarball. Those should be for devs only and devs should get the source from the repo and not the released tarball. It also makes the tarball lighter to install.